### PR TITLE
Use hyphens instead of underscores for wrench keys

### PR DESCRIPTION
**Pull request by rlhunt**
Right now it's a bit of a mess. I propose we standardize on hypens.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/999)

### CODE REVIEW

File: wrench/res/gradient.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (mrobinson) [inline wrench/res/gradient.yaml]: Did you include this file by accident?

Comment (rlhunt) [inline wrench/res/gradient.yaml]: Whoops, yeah that's for a different patch. I'll update.

**Comment by rlhunt** [comment]
Okay, I've removed the file.

**Comment by kvark** [comment]
I don't see any consensus in the world about this naming, and hyphens are easier to type, so 👍

**Comment by glennw** [comment]
@bors-servo r+

Much better!

**Pull request merged**